.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,487 @@
+            src/RealtimeServer/node_modules
+      - name: "Deps: Node"
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{matrix.node_version}}
+          cache: "npm"
+          cache-dependency-path: |
+            src/RealtimeServer/package-lock.json
+      - name: "Deps: npm"
+        env:
+          NPM_VERSION: ${{matrix.npm_version}}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:26: Double quote to prevent globbing and word splitting

(shellcheck)